index.html

```diff
@@ -37,24 +45,65 @@ <h2>I'm a Software Developer</h2>
 
         <article class="extra">
           <section class="works-centered">
```

Comment:
- [ ] I suggest to consider aligning the section to the left such that it is at the same level as the heading and the cards. This is to ensure that we stick the original Figma design ☺️.
- [ ] In addition, `project-list` should not overlap into the `p`. This makes your design messy and less user-friendly ☺️. Please fix this to make sure that your portifolio is professional 👍 
- [ ] Finally on this section, I kindly suggest to make sure that the `p` text is responsive in all the desktop screen sizes 👍 

Your design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/5b81f824-1b7c-4ee0-b934-c4862bcec367)

Figma design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/45db50e7-0545-463d-b688-82caa3abea3f)